readme.md

```diff
@@ -302,60 +218,29 @@ yarn test
 #   Requires http://entrproject.org : brew install entr
 yarn watch
 
-## run linting and unit tests separately
+## run linting, unit, and smoke tests separately
 yarn lint
 yarn unit
+yarn smoke
 
 ## run closure compiler (on whitelisted files)
 yarn closure
 ## import your report renderer into devtools-frontend and run devtools closure compiler
 yarn compile-devtools
 ```
 
-## Lighthouse as trace processor
-
-Lighthouse can be used to analyze trace and performance data collected from other tools (like WebPageTest and ChromeDriver). The `traces` and `devtoolsLogs` artifact items can be provided using a string for the absolute path on disk. The devtoolsLogs array is captured from the Network domain (a la ChromeDriver's [`enableNetwork` option](https://sites.google.com/a/chromium.org/chromedriver/capabilities#TOC-perfLoggingPrefs-object)) and reformatted slightly. As an example, here's a trace-only run that's reporting on user timings and critical request chains:
-
-### `config.json`
-
-```json
-{
-  "audits": [
-    "user-timings",
-    "critical-request-chains"
-  ],
-
-  "artifacts": {
-    "traces": {
-      "defaultPass": "/User/me/lighthouse/lighthouse-core/test/fixtures/traces/trace-user-timings.json"
-    },
-    "devtoolsLogs": {
-      "defaultPass": "/User/me/lighthouse/lighthouse-core/test/fixtures/traces/perflog.json"
-    }
-  },
-
-  "aggregations": [{
-    "name": "Performance Metrics",
-    "description": "These encapsulate your app's performance.",
-    "scored": false,
-    "categorizable": false,
-    "items": [{
-      "audits": {
-        "user-timings": { "expectedValue": 0, "weight": 1 },
-        "critical-request-chains": { "expectedValue": 0, "weight": 1}
-      }
-    }]
-  }]
-}
-```
-
-Then, run with: `lighthouse --config-path=config.json http://www.random.url`
+## Related Projects
 
-The traceviewer-based trace processor from [node-big-rig](https://github.com/GoogleChrome/node-big-rig/tree/master/lib) was forked into Lighthouse. Additionally, the [DevTools' Timeline Model](https://github.com/paulirish/devtools-timeline-model) is available as well. There may be advantages for using one model over another.
+* [webpack-lighthouse-plugin](https://github.com/addyosmani/webpack-lighthouse-plugin) - run Lighthouse from a Webpack build.
+* [lighthouse-mocha-example](https://github.com/justinribeiro/lighthouse-mocha-example) - gather performance metrics via Lighthouse and tests them in Mocha
+* [pwmetrics](https://github.com/paulirish/pwmetrics/) - gather performance metrics
+* [lighthouse-hue](https://github.com/ebidel/lighthouse-hue) - uses a Lighthouse score to set the color of Philips Hue lights
+* [lighthouse-batch](https://www.npmjs.com/package/lighthouse-batch) - run Lighthouse over a number of sites and generate a summary of their metrics/scores.
+* [lighthouse-cron](https://github.com/thearegee/lighthouse-cron) - Cron multiple batch Lighthouse audits and emit results for sending to remote server.
```

Comment:
the start of this is hard to parse, but that's what was submitted so we can leave as is for now :)